### PR TITLE
Global in sdk

### PR DESCRIPTION
**Pull request by brittcyr**
Also add tracking for balances

### CODE REVIEW

**Comment by brittcyr** [comment]
Need to add test coverage in SDK as well as consider doing the TODO in program side

File: programs/manifest/src/state/market.rs [hunk 1]
```diff
@@ -120,10 +120,21 @@ pub struct MarketFixed {
...
+    // and could be computed with other information on the market.
+    // Total base withdrawable.
+    base_withdrawable: BaseAtoms,
+    // Total quote withdrawable.
+    quote_withdrawable: QuoteAtoms,
+    // Total base in asks. Does not include global.
+    base_in_asks: BaseAtoms,
+    // Total quote in bids. Does not include global.
+    quote_in_bids: QuoteAtoms,
+
```
Comment (mschneider) [inline programs/manifest/src/state/market.rs:133]: would prefer to hide this behind a feature flag, e.g. fuzzer

Comment (brittcyr) [reply programs/manifest/src/state/market.rs:133]: Currently we have 1 market on mainnet and I plan on emptying it when rolling this out

Comment (brittcyr) [reply programs/manifest/src/state/market.rs:133]: I dont really want it for testing only. Either push it out and have it as informational for making coingecko easier, or not at all

**Comment by brittcyr** [comment]
I can split out the idl / client changes and the tracking if wanted

**Review by mschneider — APPROVED** [review]

**Pull request merged**